### ISSUE TITLE
Jira integration: supporting multiple Sites & object migration

### ISSUE THREAD

**Issue by ackernaut** [ParabolInc/parabol#9762]
### Use case
A customer has two teams that are merging. Each team has used projects on separate Jira sites. It’s a lot of work to move over tickets across sites. The team lead wants to use 2 Jira sites via the Parabol team integration in transition when estimating tickets using Poker. They want to burn through their backlog from 1 of the sites, stop using it, and move forward with the other site.
### Reproduction steps
- Issue one: the customer can’t add 2 sites as integrations on a single team
- Issue two: as a work-around the customer tried to add each site on separate teams but couldn’t
  - The customer creates team A and team B
  - The customer adds Jira site 1 to team A
  - When the customer adds Jira site 2 to team B, they only see site 1 in their Poker meeting on team B
 ### Estimated effort
- Triage what is possible, offer a next step. Can folks add more than 1 site on a single team? Can folks add different sites on different teams?

**Comment by ackernaut** [comment]
@drewhousman1010 does this make sense and look accurate? Thanks!

**Comment by Dschoordsch** [comment]
Do they have 2 sites on atlassian, similar to `site1.atlassian.net` and `site2.atlassian.net`? Do both share the same users?

**Comment by ackernaut** [comment]
> Do they have 2 sites on atlassian, similar to `site1.atlassian.net` and `site2.atlassian.net`? Do both share the same users?

Yes, he showed us 2 sites. @drewhousman1010 would you ask about membership on those sites?

**Comment by drewhousman1010** [comment]
> > Do they have 2 sites on atlassian, similar to `site1.atlassian.net` and `site2.atlassian.net`? Do both share the same users?
> 
> Yes, he showed us 2 sites. @drewhousman1010 would you ask about membership on those sites?

Just asked! Thank you both for taking charge on this.

**Comment by ScottFullGlass** [comment]
Hi All, 

Technically I have to invite users to each of the sites within our Org. That being said, all the users on our teams would be listed in each sites user list. 

Atlassian Org
- Site A (Original Site)
    - User 1, User 2, User 3 ...
- Site B (Migrated Site) 
    - User 1, User 2, User 3 ...
    
Team A: 
Team B:
    - User 1, User 2, User 3 ...

**Comment by ScottFullGlass** [comment]
In the description above, note that Site 1 was the original site under our Org at Atlassian. 
It seems like this site is taking precedence over Site 2 no matter our integration configuration. We tried removing all integrations and when connecting back to Jira only selecting Site 2. When we do this, only tickets from Site 1 appear while planning out our Poker session.

**Comment by jordanh** [comment]
Noting: this isn't a bug, we intended to only support one instance of Jira ... but this is an interesting use case to be able to have multiple Jira services connected to a single team and then be able to move objects.

Reclassifying this as an enhancement and marketing as discussion to see if we get more feedback here to prioritize how we'd design for this case.
